tests/sentry/deletions/test_grouphash.py

```diff
@@ -1,7 +1,7 @@
 from unittest.mock import patch
 from uuid import uuid4
 
-from sentry.deletions.tasks.groups import delete_groups
+from sentry.deletions.tasks.groups import delete_groups_for_project
```

Comment:
In this test, I only rename the function and pass `project_id` as a new parameter.